arduino-ide-extension/src/electron-browser/theia/core/electron-main-menu-factory.ts

```diff
@@ -178,9 +178,13 @@ export class ElectronMainMenuFactory extends TheiaElectronMainMenuFactory {
   ): Electron.MenuItemConstructorOptions[] {
     const showDisabled = options?.showDisabled !== false;
 
+    if (CompoundMenuNode.is(menu) && menu.id.includes('2_ports')) {
+      debugger;
+    }
+
     if (
       CompoundMenuNode.is(menu) &&
-      menu.children.length &&
+      (menu.children.length || alwaysVisibleSubmenu(menu)) && // hack for #569 and #655
```

Comment:
This should go upstream, without the hack
Also propose a change with the enablements

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/b0b8f7781eebdf65d035277722f8579ceaef2d0d